rotkehlchen/tests/api/test_ethereum_transactions.py

```diff
@@ -936,7 +926,6 @@ def query_transactions(rotki) -> None:
             json={
                 'from_timestamp': start_ts,
                 'to_timestamp': end_ts,
-                'exclude_ignored_assets': False,
```

Comment:
I would say we need tests for all possible scenarios. So yes most should use the default of True, but there should also be at least one that tests `exlude_ignored_assets: False` works and does what we expect

Reply:
yes should be fine